packages/core/src/utils/buildStoreUrl.ts

```diff
@@ -0,0 +1,10 @@
+export function buildStoreUrl(
+  storeUrl: string,
+  subDomainPrefix: string[]
```

Comment:
I think this will work for the majority of the cases, but if they set 
`const storeUrl = "https://www.testqa.com.br"` and then const subDomainPrefix = ["www"]. 
The result might be wrong 🤔

Reply:
Maybe we need to check if the subdomain already exists in the storeUrl before doing the replace. 
We also can use the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/hostname) constructor to parse and check it.

Reply:
As we discussed in Slack, following the [doc](https://developers.vtex.com/docs/guides/faststore/go-live-1-configuring-external-dns#step-4-associating-your-custom-domain-with-your-faststore-project) our starter should already use the subdomain in the `storeUrl` ([PR](https://github.com/vtex-sites/starter.store/pull/700) to fix this) and we can assume that the stores are providing a complete functional `storeUrl` already. Then, this function becomes unnecessary, so I've removed it.